deployment/terraform/variables.tf

```diff
@@ -588,3 +588,8 @@ variable "database_anonymizer_enabled" {
   type    = bool
   default = false
 }
+
+variable "create_bastion_iam_profile" {
```

Comment:
What is the purpose of this variable? It hasn't been used.

Reply:
It is stale code. I tested using IAM profile for the bastion host to access OpenSearch. I will remove it.

Reply:
done